src/main/java/org/apache/bcel/classfile/ConstantPool.java

```diff
@@ -232,13 +232,18 @@ public void dump( final DataOutputStream file ) throws IOException {
      * @param  index Index in constant pool
      * @return Constant value
      * @see    Constant
+     * @throws  ClassFormatException
```

Comment:
The tag description is missing.

Reply:
sorry - not sure what you mean.  There is no tag parameter to this method.

Reply:
GitHub says you added:
```
* @throws  ClassFormatException
```
The correct format is:
```
* @throws  ClassFormatException Thrown on a blue moon unless it's a leap year
```
You get the idea ;-)